en/sql/catalog.rst

```diff
@@ -882,6 +882,47 @@ A table that stores the progress status every time the **applylogdb** utility ap
 | start_time           | DATETIME        | Time when the applylogdb process accessed the slave database                                                                                       |
 +----------------------+-----------------+----------------------------------------------------------------------------------------------------------------------------------------------------+
 
+dual
+----
+
+The dual class is a one-row, one-column table that is used as a dummy table. It is used to select a constant, expression, or pseudo column such as SYS_DATE or USER. Pseudo columns can be provided as functions in CUBRID. More details and examples are in :ref:`operators-and-functions`. However, it is not mandatory to have FROM clause when selecting a constant, expression, or pseudo column because dual class will be referenced automatically. Like other system catalog classes, dual class is created to be owned by dba and dba can only execute SELECT operation. Unlike other system catalog classes, however, any user can execute SELECT operation on dual class.
+
++--------------------+---------------+----------------------------------------------------------+
+|   Attribute Name   |   Data Type   |   Description                                            |
++====================+===============+==========================================================+
+| dummy              | VARCHAR(1)    | Value used for dummy purpose only                        |
++--------------------+---------------+----------------------------------------------------------+
+
+The following example shows the result which ran the query that select pseudo column after inputting ";plan detail" or "SET OPTIMIZATION LEVEL 513;" in CSQL (:ref:`viewing-query-plan`).This shows the dual class is referenced automatically even if there is no FROM clause.
```

Comment:
1. ;plan--> ; plan (adding space between ';' and 'plan')

2. .This -->. This (adding space between '.' and 'This')

Reply:
1. ";plan" is a kind of command in CSQL, so I think it doesn't need space between ';' and 'plan' 
2. I will fix it thanks